test/postgres/providers.tf

```diff
@@ -0,0 +1,9 @@
+variable "aws_region" {
+  default = "us-west-2"
```

Comment:
my default region has been `us-east-1` as it tends to be the cheapest

Reply:
Makes sense, I was stealing from something else. :)